editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -44,6 +44,7 @@ import {
   color,
   OnClickOutsideHOC,
   iconForControlType,
+  colorTheme,
```

Comment:
there's already colorTheme imported as `colorThemeConst`, maybe it'd be worth merging those?